pkg/xds/context/context.go

```diff
@@ -6,15 +6,13 @@ import (
 	"github.com/kumahq/kuma/pkg/core"
 	core_mesh "github.com/kumahq/kuma/pkg/core/resources/apis/mesh"
 	"github.com/kumahq/kuma/pkg/core/xds"
-	"github.com/kumahq/kuma/pkg/envoy/admin"
 	"github.com/kumahq/kuma/pkg/tls"
 	"github.com/kumahq/kuma/pkg/xds/secrets"
 )
 
 type Context struct {
-	ControlPlane     *ControlPlaneContext
-	Mesh             MeshContext
-	EnvoyAdminClient admin.EnvoyAdminClient
+	ControlPlane *ControlPlaneContext
+	Mesh         *MeshContext
```

Comment:
It wasn't to avoid passing the the object. We thought this was necessary to interface with the cache. I can double check if it would work with the object itself.

Reply:
I tested this out and it works as an object. Switched it back.